docs/irq-load-balancing.md

```diff
@@ -58,4 +59,137 @@ spec:
   ... 
   runtimeClassName: performance-<profile_name>
   ...
+```
+
+## Debugging
+Here are the steps to ensure the system is configured correctly for IRQ dynamic load balancing
+
+Consider a node with 6 CPUs targeted by a 'v2' [Performance Profile](docs/performance_profile.md):
+Let's assume the node name is ```cnf-worker.demo.lab```.
+
+A profile reserving 2 CPUs for housekeeping can look like this:
+```yaml
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: dynamic-irq-profile
+spec:
+  cpu:
+    isolated: 2-5
+    reserved: 0-1
+  ...
+```
+1. Ensure you are using a v2 profile in the apiVersion.
+2. Ensure ```GloballyDisableIrqLoadBalancing``` field is missing or has the value ```false```.
+
+Start a pod configured as in [Design Details](#Design Details):
+The pod below is guaranteed and requires 2 exclusive CPUs out of the 6 available CPUs in the node.
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dynamic-irq-pod
+  annotations:
+     irq-load-balancing.crio.io: "true"
```

Comment:
can you change annotations values to `disable` one

Reply:
sure